crud/ratelimit.lua

```diff
@@ -0,0 +1,99 @@
+-- Mostly it's a copy-paste from tarantool/src/lua/log.lua. We have three
+-- reasons:
+-- 1. Tarantool has not log.crit() (a function for logging with CRIT level).
+-- 2. Only new versions of have Ratelimit type.
+-- 3. We want own copy of Ratelimit in case the implementation in Tarantool
+-- changes. Less pain between Tarantool versions.
+local ffi = require('ffi')
+
+local S_CRIT = ffi.C.S_CRIT
+local S_WARN = ffi.C.S_WARN
+
+local function say(level, fmt, ...)
+    if ffi.C.log_level < level then
+        -- don't waste cycles on debug.getinfo()
+        return
+    end
+    local type_fmt = type(fmt)
+    local format = "%s"
+    if select('#', ...) ~= 0 then
+        local stat
+        stat, fmt = pcall(string.format, fmt, ...)
+        if not stat then
+            error(fmt, 3)
+        end
+    elseif type_fmt ~= 'string' then
```

Comment:
How about commenting differences from tarantool's implementation? We don't use table as a log entry in the module -- it is completely ok to don't implement conversion to json here -- but I would leave a comment regarding removed parts of the implementation to decrease possible confusion.

Reply:
Great idea, done.